src/pages/Dashboard/components/AccountList/AccountList.tsx

```diff
@@ -24,14 +24,16 @@ export const AccountList = () => {
   const accountRows = useMemo(() => {
     return (
       <>
-        {Object.values(balances).map(account => (
-          <AccountRow
-            key={account.contract ?? account.chain}
-            balance={account.balance ?? '0'}
-            chain={account.chain}
-            tokenId={account.contract}
-          />
-        ))}
+        {Object.values(balances)
+          .sort((a, b) => ((a?.balance ?? '0') > (b?.balance ?? '0') ? -1 : 1))
```

Comment:
I am wondering if this will not work quite right due to balance or 0 being string values compared. I think you could use `(b?.balance ?? '0') - (a?.balance ?? '0')` to ensure they are treated as numbers in all cases instead?

Reply:
"420" > "69" = false is the problem with string comparison like this.

Reply:
personally id use bignumber comparisons for this